x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/settings_page/output_section.tsx

```diff
@@ -6,12 +6,13 @@
  */
 
 import React from 'react';
-import { EuiTitle, EuiText, EuiSpacer, EuiButtonEmpty } from '@elastic/eui';
+import { EuiTitle, EuiText, EuiSpacer, EuiButtonEmpty, EuiLink } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 
 import { useLink } from '../../../../hooks';
 import type { Output } from '../../../../types';
 import { OutputsTable } from '../outputs_table';
+import { useStartServices } from '../../hooks';
```

Comment:
@jlind23  - it seems this is causing a typescript error. `../../hook` doesn't indeed export this variable.

It's also logged in the type errors in the build:
```
proc [tsc] x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/settings_page/output_section.tsx:15:10 - error TS2305: Module '"../../hooks"' has no exported member 'useStartServices'.
--
  | proc [tsc]
  | proc [tsc] 15 import { useStartServices } from '../../hooks';


```

Reply:
Probably you could import it from the other hooks: `import { useLink } from '../../../../hooks';`

Reply:
Thanks @delanni, just pushed a fix.